tests/test_cli_convertor.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2022 spdx tool contributors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from unittest import TestCase
+
+from spdx.cli_tools.convertor import determine_infile_and_outfile
+
+from tests.testing_utils import raises
+
+class TestConvertor(TestCase):
+    maxDiff = None
+
+    def test_determine_input_with_known_i_o_format(self):
+        infile_given = 'infile.rdf'
+        outfile_given = 'outfile.json'
+        src = ()
+        from_ = None
+        to = None
+
+        infile, outfile = determine_infile_and_outfile(infile_given, outfile_given, src, from_, to)
+
+        assert infile == infile_given
+        assert outfile == outfile_given
+
+    def test_determine_input_with_unknown_i_o_format(self):
+        infile_given = None
+        outfile_given = None
+        src = ('infile.in', 'outfile.out')
+        from_ = 'rdf'
+        to = 'json'
+
+        infile, outfile = determine_infile_and_outfile(infile_given, outfile_given, src, from_, to)
+
+        infile_path = os.path.splitext(src[0])[0]
+        infile_given = infile_path + "." + from_
+        outfile_path = os.path.splitext(src[1])[0]
+        outfile_given = outfile_path + "." + to
```

Comment:
This is basically a reimplementation of the logic that we are testing. For tests, I would suggest just statically providing the expected result and asserting on it (e.g. including `expected_infile_path = infile.rdf`)

Reply:
(the same applies to the other test cases, of course)